packages/rancher-istio/charts/templates/istio-install-job.yaml

```diff
@@ -2,27 +2,29 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: istioctl-installer
-  namespace: istio-system
+  namespace: {{ .Values.global.istioNamespace | default "istio-system" }}
   annotations:
     "helm.sh/hook": post-install,post-upgrade
     "helm.sh/hook-weight": "-5"
-    # Todo: use following line instead
-    # "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
-    "helm.sh/hook-delete-policy": before-hook-creation
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
 spec:
   backoffLimit: 1
   template:
     spec:
       containers:
         - name: istioctl-installer
-          image: rancher/istio-installer:{{ .Values.installerVersion }}
+          image: brendarearden/istio-test:4.1.6
```

Comment:
will update when new rancher/istio-installer is available prior to merging this pr

Reply:
just a reminder to do this